src/contexts/current-content-type.tsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, ReactNode, useContext } from 'react'
+
+type CurrentContentType = 'global' | 'docs' | 'tutorials'
+
+const CurrentContentTypeContext = createContext<CurrentContentType>(undefined)
+
+interface CurrentContentTypeProviderProps {
+	children: ReactNode
+	currentContentType: CurrentContentType
+}
+
+const CurrentContentTypeProvider = ({
```

Comment:
suggestion: A brief doc string here explaining what this provider is for would be very valuable 🙏

Reply:
Can do!

Reply:
Added a block comment in https://github.com/hashicorp/dev-portal/pull/953/commits/efe66407da5f8fd65c48200b1205ee2cf853b454 ✅